src/app/modules/users/single-user/single-user.component.scss

```diff
@@ -23,3 +23,6 @@
     margin-top: 9%;
     margin-right: 15%;
 }
+.mat-button {
```

Comment:
Please don't override the material library classes

Reply:
changed